onediff_diffusers_extensions/diffusers_extensions/deep_cache/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .models.pipeline_utils import disable_deep_cache_pipeline
```

Comment:
models is missing an `__init__.py`:

```
  File "/root/.cache/isolate/virtualenv/17f872daadbd2c97c73b5214344b569a73364eb29c53f23c0d51d9216649d4d5/lib/python3.11/site-packages/diffusers_extensions/deep_cache/__init__.py", line 1, in <module>
ModuleNotFoundError: No module named 'diffusers_extensions.deep_cache.models'
```

happens when trying to import it like this:
```
from diffusers_extensions.deep_cache import StableDiffusionPipeline
```

Reply:
> models is missing an `__init__.py`:
> 
> ```
>   File "/root/.cache/isolate/virtualenv/17f872daadbd2c97c73b5214344b569a73364eb29c53f23c0d51d9216649d4d5/lib/python3.11/site-packages/diffusers_extensions/deep_cache/__init__.py", line 1, in <module>
> ModuleNotFoundError: No module named 'diffusers_extensions.deep_cache.models'
> ```
> 
> happens when trying to import it like this:
> 
> ```
> from diffusers_extensions.deep_cache import StableDiffusionPipeline
> ```

Thanks for the suggestion. We tested it in our own environment, but it may not be enough. Your suggestions will make it compatible with more environments.